max_lin_speed: 3.0
lin_step: 0.25

max_ang_speed: 3.0
ang_steps: 500

dead_man_index: 5